manager/gui/src/lib/components/SgtCard/SgtCard.vue

```diff
@@ -64,6 +64,7 @@ export default class SgtCard extends Vue {
   align-items: center;
   justify-content: space-between;
   margin-bottom: 12px;
+  font-weight: bold !important;
```

Comment:
Try to make it work without using '!important'. Can use '::v-deep' instead I guess.

Reply:
Important is required, since there is an important used by vuetify in their class.